library/src/MetricsReporter.ts

```diff
@@ -1,35 +1,46 @@
 import { AggregatedStats } from '@epicgames-ps/lib-pixelstreamingfrontend-ue5.4';
 import { v4 as uuidv4 } from 'uuid';
 
-declare var METRICS_URL: string;
-
-const SupportedStats : Record<string, string> = {
-	'video_width': 'Stream resolution width in pixels.',
-	'video_height': 'Stream resolution height in pixels.',
-	'video_bitrate': 'Stream video bitrate in bits per second.',
-	'video_dropped': 'Video stream frames dropped.',
-	'video_packets_lost': 'Video stream packets lost',
-	'video_fps': 'Video stream frames per second.',
-	'video_pli_count': 'Picture Loss Indication count.',
-	'video_keyframes': 'Video stream keyframes.',
-	'video_nack_count': 'Video stream negative acknowledgements.',
-	'video_freeze_count': 'Video stream freeze count.',
-	'video_jitter': 'Video stream jitter.',
-	'video_frame_count': 'Video stream frame count.',
-	'audio_bitrate': 'Audio stream bitrate in bits per second.',
-	'loading_time': 'Stream loading time in milliseconds.',
-	'session_duration': 'Stream duration in milliseconds.'
+declare var BUCCANEER_URL: string;
+
+enum StatOperation {
+	Reset = 1,
+	Add,
+	Average
```

Comment:
Would be good have stat operations for min, max, stddev too.

Reply:
I am thinking of bitrate for example where we would be interested in knowing not just the average but the best/worst and spread too.